frontend/www/js/omegaup/components/submissions/List.vue

```diff
@@ -16,19 +24,37 @@
           }}
         </h3>
       </div>
-      <div class="panel-body" v-if="shouldShowControls">
-        <template v-if="page > 1">
-          <a class="prev" v-bind:href="`/submissions?page=${page - 1}`">
-            {{ T.wordsPrevPage }}</a
+      <div class="panel-body" v-if="includeUser || showControls">
+        <template v-if="includeUser">
+          <label
+            ><omegaup-autocomplete
+              class="form-control"
+              v-bind:init="el =&gt; UI.userTypeahead(el)"
+              v-model="searchedUsername"
+            ></omegaup-autocomplete
+          ></label>
+          <button
+            class="btn btn-primary"
+            type="button"
+            v-on:click="onSelectUser"
           >
-          <span class="delimiter" v-show="shouldShowNextPage">|</span>
+            {{ T.searchUser }}
+          </button>
+        </template>
+        <template v-if="showControls">
+          <template v-if="page > 1">
+            <a class="prev" v-bind:href="`/submissions?page=${page - 1}`">
```

Comment:
nit:

```suggestion
            <a class="prev" v-bind:href="`/submissions/?page=${page - 1}`">
```

siempre que puedas poner el trailing slash, ponlo. Igual en L54.